packages/app-accounts/src/modals/Create.tsx

```diff
@@ -229,26 +229,53 @@ function Create ({ className, onClose, onStatusChange, seed: propsSeed, type: pr
           <Input
             autoFocus
             className='full'
-            help={t('Name given to this account. You can edit it. To use the account to validate or nominate, it is a good practice to append the function of the account in the name, e.g "name_you_want - stash".')}
             isError={!isNameValid}
-            label={t('name')}
+            label={t('Name your account (you can edit later)')}
             onChange={_onChangeName}
             onEnter={_onCommit}
-            placeholder={t('new account')}
+            placeholder={t('e.g. validate, nominate')}
```

Comment:
I'm not sure these are very good example names 🤷‍♂️

Reply:
this was as per design specification 😳